integration/graphql-type-factories.ts

```diff
@@ -7,13 +7,15 @@ import {
   Child,
   PopularityDetail,
   SaveAuthorResult,
+  SearchResults,
   WorkingDetail,
   Named,
   Popularity,
   Working,
 } from "./graphql-types-only";
 import { newDate } from "./testData";
 
+const factories: Record<string, Function> = {};
```

Comment:
This file has a lot of updates that should have been in the last PR, but the `npm run graphql-codegen` only updated the other file; now it updates both, so these changes are getting this file caught up.